en/docs/develop-components/develop-services/develop-a-grpc-service.md

```diff
@@ -0,0 +1,210 @@
+2. On the **Components** page, click **Create**.
+3. On the **Manual Trigger** card, click **Create**.
+4. Enter a unique name and a description of the client component. For this guide, let's enter the following values:
+
+    | **Field**       | **Value**               |
+    |-----------------|-------------------------|
+    | **Name**        | `Go gRPC Client`        |
+    | **Description** | `Receive greetings`     |
+
+5. Click **Next**.
+6. To allow Choreo to connect to your GitHub account, click **Authorize with GitHub**.
+7. Select the following values to connect the repository:
+
+    | **Field**             | **Description**                 |
+    |-----------------------|---------------------------------|
+    | **GitHub Account**    | Your account                    |
+    | **GitHub Repository** | **`choreo-sample-apps`**        |
+    | **Branch**            | **`main`**                      |
+    | **Build Preset**      | **Dockerfile**                  |
+    | **Dockerfile path**   | **go/grpc/Dockerfile.client** |
+    | **Docker context path** | **go/grpc**                   |
+
+8. Click **Create** . Once the component creation is complete, you will see the component overview page.
+
+### Step 4.2: Setup environment variables
+
+The client application, in this case, the gRPC client,  needs the server URL of the gRPC server service. This is read from the client application as an environment variable. Follow the steps below to configure the environment variable for the client application:
+
+1. Navigate to the **gRPC Server** component's **DevOps** page from the left navigation and click on **Configs and Secrets**.
+3. Click **+ Create**.
+4. Select **Config Map** as the **Config Type** and **Environment Variables** as the **Mount Type**.
+5. Click **Next**.
+6. Provide the Following values to add the environment variables:
+
+    | **Field**       | **Value**               |
+    |-----------------|-------------------------|
+    | **Config Name** | `serviceurl`            |
+    | **Name**        | `GREETER_SERVICE`       |
+    | **Value**       | Paste the URL value copied from the gRPC service in the previous step of the guide.  Make sure to **drop the `http://` or `grpc://` part and the trailing `/` from the URL when copying**. For example, add only `go-grpc-greeter-3192360657:8080`.|
+
+7. Click **Create**.
+
+### Step 4.3: Build and deploy the gRPC client component
+
+Now that we have connected the source repository, and configured the environment variable details, let's build and run the greeter client.
+
+To build and run the client, follow these steps:
+
+1. Go to the **Deploy** page and click **Deploy Manually**.
+2. Click **Deploy Manually**.
+3. Now let's trigger the gRPC client.  On the **Development** environment card click **Run Once**,  multiple times. This will generate logs.
+4. Navigate to the **Observability** page and view **Logs**.
```

Comment:
Shall we mention how to navigate to the Observability page?
In the left navigation menu, click **Observability**.